scenarios/scenario2/variables.tf

```diff
@@ -12,6 +7,6 @@ variable "ecs_image" {}
 variable "scenario" {}
 variable "psql_version" {}
 variable "psql_port" {}
-variable "psql_password" {}
+variable "psql_password" { default = "qwerty"}
```

Comment:
`qwerty` won't be valid, there are password sec restrictions, so creation of RDS will fail

Reply:
qwerty12345! will fit